### PR TITLE
Bugfix: Add object counts to arangoimport latency 

### PR DESCRIPTION
**Pull request by matthewvon**
This is a back port of PR 9856 (3.4).

Existing latency report stated number of requests posted to db server. But there was not tracking of how many objects / records were contained in each request. This PR utilizes the unused statistic location and populates it with total count of objects within the requests.

### CODE REVIEW

**Comment by matthewvon** [comment]
Jenkins running:  http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/6100/

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* Add count of objects to latency reporting in arangoimport
```
Comment (jsteemann) [inline CHANGELOG]:
```suggestion
* Add count of objects to latency reporting in arangoimport.
```

File: arangosh/Import/QuickHistogram.h [hunk 2]
```diff
@@ -49,12 +49,14 @@ class QuickHistogram : public arangodb::Thread {
...
   QuickHistogram& operator=(QuickHistogram const&) = delete;
 
- public:
-  QuickHistogram()
-      : Thread("QuickHistogram"),
-        _writingLatencies(nullptr),
-        _readingLatencies(nullptr),
-        _threadRunning(false) {}
+public:
+QuickHistogram()
```
Comment (jsteemann) [inline arangosh/Import/QuickHistogram.h]: wrong indentation

**Comment by jsteemann** [comment]
In addition, it would be nice if this PR removed the unused `totalRead` variable from arangosh/Import.

**Comment by matthewvon** [comment]
The commenting out of totalRead was someone else’s change. I did not know if it was temporary or permanent @jsteemann

**Comment by jsteemann** [comment]
Why did you copy "someone else's change" into the 3.5 port of the PR then?
Anyway, the `totalRead` variable is not used. It is only initialized and incremented, but never read back.
So it can be removed altogether, in both 3.5 and devel.
3.4 can stay as it is, as I don't think we should do active code cleanup there given most development happens in more recent branches.

**Comment by Simran-B** [comment]
@matthewvon Does this need to be updated in the docs? https://www.arangodb.com/docs/stable/programs-arangoimport-details.html#result-output

**Comment by matthewvon** [comment]
@Simran-B There is currently no documentation about latency option output.  I leave the decision for whether or not there should be documentation to the Cologne team to discuss.  It was originally designed solely as a debug tool.

**Comment by matthewvon** [comment]
@jsteemann I believe lastest commit addresses all code review issues.

**Comment by matthewvon** [comment]
Jenkins running:  http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/6119/

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/10136/

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/10143/

**Pull request merged**